src/data/index.js

```diff
@@ -0,0 +1,9 @@
+const buildH1Data = require("./h1");
+
+function buildData(invaderStructDefs) {
+  return {
+    h1: buildH1Data(invaderStructDefs)
```

Comment:
Should be easily extensible with H2 tags in the future, using whatever external dependencies we need to (I think OpenSauce has some H2 tag definitions in XML).

Reply:
We can grab them from tools like assembly. Or extract them directly from their h2hek.

For them the assembly approach might be the best though, as that is what they use the most.